common/hubserver.go

```diff
@@ -128,6 +128,10 @@ func (h *hubServer) Logout() bool {
 	urlStr := fmt.Sprintf("%v", u)
 
 	resp, err := h.client.Get(urlStr)
+	if err != nil {
+		log.Printf("ERROR client.Get %s\n", err)
```

Comment:
%+v is probably more general for these sorts of prints iirc as a code convention?

Reply:
%v is usually best